input.js

```
@@ -0,0 +1,31 @@
+    console.error("Error:", err);
+    return;
+  }
+  console.log("Deleted:", stdout);
+});
+
+// 🔥 2. Operación bloqueante (sincrónica)
+const config = fs.readFileSync('./config.json');  // ❌ Debe ser async
+console.log("Config loaded:", config);
+
+// 🔥 3. Uso de eval
+eval("console.log('Esto es muy inseguro')");  // ❌ Nunca usar eval
```

Biome (1.9.4)

[error] eval() exposes to security risks and performance issues.

See the MDN web docs for more details.
Refactor the code so that it doesn't need to call eval().

(lint/security/noGlobalEval)